capsules/src/net/thread/tlv.rs

```diff
@@ -1334,7 +1334,7 @@ impl NetworkManagementTlv<'a> {
                     NetworkManagementTlv::ActiveTimestamp {
                         timestamp_seconds: timestamp_seconds,
                         timestamp_ticks: timestamp_ticks >> 1,
-                        u_bit: (timestamp_ticks | 1u16) > 0,
+                        u_bit: true,
```

Comment:
```suggestion
                        u_bit: (timestamp_ticks & 1u16) > 0,
```

I don't think `true` is quite right. From skimming around the code, for whatever reason, it looks like this TLV stores `timestamp_ticks` with one less bit of precision, that lost bit is kept around in this `u_bit` field, and then later it's restored when re-encoding this TLV. I think it was trying to pick off the last bit from that field (and the `>0` was a technique to coerce to a `bool` to make Rust happy)